frontend/packages/knative-plugin/integration-tests/support/step-definitions/eventing/eventing-create-sink-broker.ts

```diff
@@ -0,0 +1,117 @@
+import { When, Then, Given } from 'cypress-cucumber-preprocessor/steps';
+import { devNavigationMenu } from '@console/dev-console/integration-tests/support/constants';
+import { eventSourcePO } from '@console/dev-console/integration-tests/support/pageObjects';
+import { navigateTo } from '@console/dev-console/integration-tests/support/pages';
+import { topologyPage } from '@console/topology/integration-tests/support/pages/topology';
+import { eventingPO } from '../../pageObjects/global-po';
+
+Given('user selects Broker from Developer Catalog card', () => {
+  cy.get(eventingPO.brokerCard).click();
+});
+
+Given('user selects Event Source from Developer Catalog card', () => {
+  cy.get(eventingPO.eventSourceCard).click();
+});
+
+Then('user will see Form view default selected', () => {
+  cy.get(eventingPO.formView).should('exist');
+});
+
+Then('user will see YAML view radio button', () => {
+  cy.get(eventingPO.yamlView).should('exist');
+});
+
+Then('user will see Application field for broker', () => {
+  cy.byLegacyTestID('application-form-app-input').should('exist');
+});
+
+Then('user will see Name field for broker', () => {
+  cy.byLegacyTestID('application-form-app-name').should('exist');
+});
+
+When('user selects YAML view', () => {
+  cy.get(eventingPO.yamlView).should('exist');
+});
+
+When('user clicks on Create button to create', () => {
+  cy.get(eventingPO.submit).click();
+});
+
+Then('user will be redirected to Topology page', () => {
+  topologyPage.verifyTopologyPage();
+});
+
+Then('user will see the {string} broker created', (brokerName: string) => {
+  topologyPage.verifyWorkloadInTopologyPage(brokerName);
+});
+
+Given('user has created broker {string}', (brokerName: string) => {
+  navigateTo(devNavigationMenu.Topology);
+  topologyPage.verifyWorkloadInTopologyPage(brokerName);
+});
```

Comment:
If a broker is not present in the topology then you should provide a facility to create one